### ISSUE TITLE
`dvb` doesn't work as expected

### ISSUE THREAD

**Issue by mklinga** [VSCodeVim/Vim#1095]
Please *thumbs-up* 👍 this issue if it personally affects you! You can do this by clicking on the emoji-face on the top right of this post. Issues with more thumbs-up will be prioritized.

-----

### What did you do?

Pressed the keys `dvb` on the normal mode.

### What did you expect to happen?

Delete word backwards with current character included.

### What happened instead?

On the normal mode, pressing `dvb` does only remove the character under the cursor and then moves backward a word.

### Technical details:

* VSCode Version: 1.7.2
* VsCodeVim Version: 0.4.3
* OS: macOS Sierra

**Comment by johnfn** [comment]
Looks like a bug to me.

**Comment by Chillee** [comment]
Could you provide some documentation or explanation about what `dvb` is supposed to do? I'm having a hard time finding information on what v is supposed to do here.

**Comment by mklinga** [comment]
Shortly - it should be equal to `vbd`. As in: remove `word` backwards from the cursor with the current character included.

**Comment by mklinga** [comment]
In reality the `v` modifier seems to be a bit more complex, see `:help o_v` in the vim docs. My usecase that I reported refers to the later paragraph:
```
If the motion already was characterwise, toggle
inclusive/exclusive.  This can be used to make an exclusive
motion inclusive and an inclusive motion exclusive.
```

**Comment by martinhj** [comment]
Yes, I really miss this and I think https://github.com/vim/vim/blob/202d982b36d87cf91d992bd7e30d3223bdc72cd9/runtime/doc/motion.txt#L118-L131 does a good job describing the functionality and the benefits of altering the motion to be 'inclusive' or 'exclusive'.

**Comment by adamschoenemann** [comment]
Miss this too!

**Comment by mferber** [comment]
I'm voting for this one too. I was surprised to discover that this motion modifier isn't supported. (At least there's the visual-mode workaround.)

**Comment by 20jasper** [comment]
Commenting for visibility!
